docs/products/kafka/karapace.rst

```diff
@@ -0,0 +1,44 @@
+Karapace
+========
+What is Karapace?
+-----------------
+Karapace is an Aiven built open-source Schema Registry for Apache Kafka®, and provides a central repository to store and retrieve schemas. It consists of a Schema Registry and a REST API. All Kafka services on Aiven support both these features (Schema registry and REST API), and as a user, based on your requirements, you can enable or disable them. 
```

Comment:
Should we link to https://karapace.io/ in the intro?

Reply:
Added link in the into.